library/std/intrinsic.qs

```diff
@@ -328,7 +328,7 @@ namespace Microsoft.Quantum.Intrinsic {
     }
 
     /// # Summary
-    /// Applies a rotation about the $\ket{1}$ state by a given angle.
+    /// Applies a rotation about the |1⟩ state by a given angle.
```

Comment:
```suggestion
    /// Applies a rotation about the _|1⟩_ state by a given angle.
```

Reply:
I think it makes sense to italicize ket notation instances.

Reply:
I looked at it in VSCode rendition and it doesn't look very good in italic... This is subjective, of course.

Reply:
I think I agree that italics look a bit strange for the ket notation. @cesarzc what do you think?

Reply:
It does look a bit strange. I don't feel strongly about this so not italicizing ket notation is fine too. It just feels slightly inconsistent but not a big deal imo.

Reply:
Let's go without italicizing it for now then. Thanks for the feedback!